ingestion-beam/src/main/java/com/mozilla/telemetry/contextualservices/ParseReportingUrl.java

```diff
@@ -34,13 +36,17 @@
  * Extract reporting URL from document and filter out unknown URLs.
  */
 public class ParseReportingUrl extends
-    PTransform<PCollection<PubsubMessage>, Result<PCollection<PubsubMessage>, PubsubMessage>> {
+    PTransform<PCollection<PubsubMessage>, Result<PCollection<SponsoredInteraction>, PubsubMessage>> {
 
   private final String urlAllowList;
 
   private static transient Set<String> singletonAllowedImpressionUrls;
   private static transient Set<String> singletonAllowedClickUrls;
 
+
+  private static final String NS_DESKTOP = "contextual-services";
+  private static final String NS_FENIX = "org.mozilla.fenix";
```

Comment:
Note that the appIds use dots, but the corresponding namespaces use hyphens. So the expected namespaces here as they appear in PubSub attributes should be `org-mozilla-fenix`, `org-mozilla-firefox-beta`, and `org-mozilla-firefox`.

Reply:
I am removing the FENIX namespace here as its not currently used.